File: app/src/main/java/com/zillennium/utswap/screens/systems/notificationScreen/NotificationActivity.java
package com.zillennium.utswap.screens.systems.notificationScreen;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;


import com.zillennium.utswap.R;
import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivitySystemNotificationBinding;

import java.util.ArrayList;

public class NotificationActivity extends AppCompatActivity {

    ActivitySystemNotificationBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivitySystemNotificationBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        ArrayList arrayList = new ArrayList();
        for (int i = 0; i < 20; i++) {
            arrayList.add(new Notification(R.drawable.ut5, "Complete Register", "Zillion United welcomes you to our trading platform. Your trading account is being reviewed by our admin. Please be patient", "6 day ago"));
        }
        RecyclerView recyclerView = this.binding.rvNotification;
        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        recyclerView.setAdapter(new NotificationAdapter(arrayList));

        
        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(NotificationActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/system/notificationScreen/NotificationActivity.kt
package com.zillennium.utswap.screens.system.notificationScreen

import androidx.recyclerview.widget.LinearLayoutManager
import com.zillennium.utswap.R
import com.zillennium.utswap.UTSwapApp
import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.databinding.ActivitySystemNotificationBinding
import java.io.IOException

class NotificationActivity :
    BaseMvpActivity<NotificationView.View, NotificationView.Presenter, ActivitySystemNotificationBinding>(),
    NotificationView.View {

    override var mPresenter: NotificationView.Presenter = NotificationPresenter()
    override val layoutResource: Int = R.layout.activity_system_notification

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                val arrayList = ArrayList<Notification>()
                for (i in 0..19) {
                    arrayList.add(
                        Notification(
                            R.drawable.ut5,
                            "Complete Register",
                            "Zillion United welcomes you to our trading platform. Your trading account is being reviewed by our admin. Please be patient",
                            "6 day ago"
                        )
                    )
                }
                rvNotification.layoutManager = LinearLayoutManager(UTSwapApp.instance)
                rvNotification.adapter = NotificationAdapter(arrayList)

                // Set Passed Back
                ivBack.setOnClickListener {
                    onBackPressed()
                }

            }
            // Code
        } catch (error: Exception) {
            // Must be safe
        }
    }
}